tests/testthat/test-survival-augment.R

```diff
@@ -0,0 +1,89 @@
+library(testthat)
+library(tidymodels)
+library(prodlim)
+library(censored)
+
+
+# ------------------------------------------------------------------------------
+
+test_that('augmenting survival models ', {
+  skip_if_not_installed("parsnip", minimum_version = "1.1.0.9001")
+
+  set.seed(1)
+  sim_dat <- SimSurv(500) %>%
+    mutate(event_time = Surv(time, event)) %>%
+    select(event_time, X1, X2)
+
+  set.seed(2)
+  split <- initial_split(sim_dat)
+  sim_tr <- training(split)
+  sim_te <- testing(split)
```

Comment:
Since this is not relevant for testing `augment()`, I would remove it here to keep the tests focussed on the essentials.